internal/cli/down.go

```diff
@@ -80,25 +82,55 @@ func (c *downCmd) down(ctx context.Context, downDeps DownDeps, args []string) er
 		return err
 	}
 
-	var dcProject model.DockerComposeProject
+	var dcServices []model.DockerComposeUpSpec
 	for _, m := range tlr.Manifests {
 		if m.IsDC() {
-			dcProject = m.DockerComposeTarget().Spec.Project
-			break
+			dcServices = append(dcServices, m.DockerComposeTarget().Spec)
 		}
 	}
 
-	if !model.IsEmptyDockerComposeProject(dcProject) {
+	if len(dcServices) > 0 {
 		dcc := downDeps.dcClient
-		err = dcc.Down(ctx, dcProject, logger.Get(ctx).Writer(logger.InfoLvl), logger.Get(ctx).Writer(logger.InfoLvl))
+		// at the time of writing, a Tiltfile can only have one DC project
+		proj := dcServices[0].Project
+
+		// `docker-compose down` removes all services in the config
+		// if we are only stopping a subset, we use `docker-compose rm` instead
+		// (don't *always* use `rm` because it doesn't remove networks)
+		useDown, err := isAllServicesInProject(ctx, dcc, proj, dcServices)
 		if err != nil {
-			return errors.Wrap(err, "Running `docker-compose down`")
+			return err
+		}
+		if useDown {
+			err := dcc.Down(ctx, proj, logger.Get(ctx).Writer(logger.InfoLvl), logger.Get(ctx).Writer(logger.InfoLvl))
+			if err != nil {
+				return errors.Wrap(err, "Running `docker-compose down`")
```

Comment:
i think FormatError in the Down() method already does this wrapping? feels redundant